### PR TITLE
Fix a load of link errors across the site

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Fix all links on live

## QA
- Check all the new links work
- Check it covers all errors reported by [the latest links on live action](https://github.com/canonical-web-and-design/ubuntu.com/runs/2525242958?check_suite_focus=true)

### CODE REVIEW

File: templates/pricing/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% block content %}
   <section class="p-strip--suru-topped">
     <div class="u-fixed-width">
-      <h1>Plans &amp; pricing</h1>
+      <h1>Plans andf pricing</h1>
```
Comment (fitojb) [inline templates/pricing/index.html:11]: tiny typo here ;)

Comment (pmahnke) [inline templates/pricing/index.html:11]: I don't think you meant this.

**Review by pmahnke — APPROVED** [review]
one typo, otherwise good job!

**Comment by anthonydillon** [comment]
Thanks @fitojb and @pmahnke, good spot :+1:

**Pull request merged**